### ISSUE TITLE
Error detected while processing function <SNR>31_BEActivateBuffer: line 18: E684: list index out of range: 2

### ISSUE THREAD

**Issue by alphaCTzo7G** [amix/vimrc#426]
Hi all,

I am getting this error in my VIM recently. Has anybody faced this issue? if so how did you resolve it?

It appears to be happening because of `buffexplorer`, because `BEActivateBuffer` is a function in the `.vim` file in the `buffexplorer` plugin.

But line 18 is a comment in that file. so I am unable to understand what is going on here. 

Any idea about what E684 stands for? Or how to resolve this? The only way I can resolve this issue is to remove BuffExplorer altogether, and start using `CtrlpBuffer`, but `Ctrlp` doesn't provide a easy way to open buffers in vertical splits, or in horizontal splits..'

https://github.com/jlanzarotta/bufexplorer/issues/72

**Comment by alphaCTzo7G** [comment]
This appears to be happening due to a clash between BufExplorer and restore_view.vim plugin. 

To reproduce this issue:

1. open a lot of files simultaneously: my test was done with 7 python, md, txt, conf files open simultaneously.
2. switch open these files in different tabs, or vertical splits
3. while switching the error: `Error detected while processing function <SNR>31_BEActivateBuffer: line 18: E684: list index out of range: 2` will appear, if `restore_view.vim` and `bufexplorer.vim` are simultaneously loaded in the load path

**Comment by alphaCTzo7G** [comment]
For now, I have replaced `buffexplorer` in my plugins with the following:

> map <leader>o :CtrlPBuffer<cr>
> map <leader>be :CtrlPBuffer<CR>
> map <leader>bs :sp<CR>:CtrlPBuffer<CR>
> map <leader>bv :vs<CR>:CtrlPBuffer<CR>

And removed `buffexplorer`. Buffexplorer was amazing with its simple interface but perhaps this is a bug which maynot be resolved soon. So, this pretty much provides the same functionality that `buffexplorer` and this vimrc provides but with `ctrlp`

**Comment by alphaCTzo7G** [comment]
Seems like this is because of a old version of `bufexplorer`. The current version of `bufexplorer` is from 2010, v7.2.8. While the latest version is 7.4.10. That has removed the `BEActivateBuffer` function.. 

I am using a slightly older version of `amix/vimrc`.. but I checked with the latest version of this `amix/vimrc` and updated the plugins using `python3 update_plugins.py`, but it seems to still use `v7.2.8` version of `bufexplorer`.

Since, `restore_view` is pretty much the only plugin that restores views for filetypes that are manually folded.. it would helpful to those users who rely on manual folding. 

I tried out the latest version of `bufexplorer` and it didnt cause any issues so far, and its probably better to update the `bufexplorer` plugin since the current version is 8 years old.. since then a lot of improvements have been made to `bufexplorer`. Is is possible to update `buffexplorer`?
